agences-regionales-sante/nouvelle-aquitaine/2020-03-18.yaml

```diff
@@ -0,0 +1,48 @@
+date: "2020-03-18"
+source:
+  nom: ARS Nouvelle-Aquitaine
+  url: https://www.nouvelle-aquitaine.ars.sante.fr/system/files/2020-03/CP_Covid19_ARSNA_18_03_2020_0.pdf
+  archive: https://web.archive.org/web/20200318211955/https://www.nouvelle-aquitaine.ars.sante.fr/system/files/2020-03/CP_Covid19_ARSNA_18_03_2020_0.pdf
+donneesRegionales:
+  nom: Nouvelle-Aquitaine
+  code: REG-75
+  casConfirmes: 242
+  gueris: 2
```

Comment:
A supprimer, le nombre de guéris n'est pas noté dans le bulletin :)

Reply:
Done.

Reply:
Oh wait, trompée :')